worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -91,7 +93,7 @@ namespace RTC
 			{
 				// Calculate how many elements would it be necessary to add when pushing new item
 				// to the back of the deque.
-				auto addToBack = static_cast<uint16_t>(seq - (this->startSeq + this->buffer.size() - 1));
+				auto addToBack = static_cast<uint16_t>(seq - (this->startSeq + bufferSize - 1));
```

Comment:
I don't see that we are checking whether the adding blank slots won't exceed MaxSeq.